### PR TITLE
[Merged by Bors] - feat(Subgroup): apply_ofInjective_symm

### PR DESCRIPTION
**Pull request by ChrisHughes24**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/GroupTheory/Subgroup/Basic.lean [hunk 1]
```diff
@@ -2764,6 +2764,11 @@ theorem ofInjective_apply {f : G →* N} (hf : Function.Injective f) {x : G} :
 #align monoid_hom.of_injective_apply MonoidHom.ofInjective_apply
 #align add_monoid_hom.of_injective_apply AddMonoidHom.ofInjective_apply
 
+@[to_additive (attr := simp)]
+theorem apply_ofInjective_symm {f : G →* N} (hf : Function.Injective f) (x : f.range) :
+    f ((ofInjective hf).symm x) = x := by
+  rw [← ofInjective_apply hf, MulEquiv.apply_symm_apply]
```
Comment (eric-wieser) [inline Mathlib/GroupTheory/Subgroup/Basic.lean:2770]: Optional golf:
```suggestion
    f ((ofInjective hf).symm x) = x := 
  Subtype.ext_iff.1 <| (ofInjective hf).apply_symm_apply x
```
to match the proof for `Equiv.apply_ofInjective_symm`

bors d+

**Comment by ChrisHughes24** [comment]
bors merge

**Pull request closed without merging**